diff --git a/compute-prefix.sh b/compute-prefix.sh
index f38105d..153864e 100755
--- a/compute-prefix.sh
+++ b/compute-prefix.sh
@@ -24,7 +24,7 @@ if (( $? != 0 )); then
     exit 4
 fi
 
-main_loc=$(dirname $bashdb_main)
+main_loc=$(dirname ${bashdb_main##*\'})
 
 # Strip expected suffixes we would find in main_loc:
 # In particular <prefix>/share/bashdb/ -> <prefix>
diff --git a/configure.ac b/configure.ac
index cf96d6f..89cbfda 100644
--- a/configure.ac
+++ b/configure.ac
@@ -21,7 +21,7 @@ dnl Process this file with autoconf to produce a configure script.
 
 define(DEBUGGER, bashdb)
 define(POSIXSHELL, bash)
-define(OK_BASH_VERS, 5.2)
+define(OK_BASH_VERS, 5.3)
 define(relstatus, 1.1.2)
 
 AC_INIT([bashdb],[OK_BASH_VERS-relstatus],[https://sourceforge.net/p/bashdb/bugs/new/])
@@ -103,7 +103,7 @@ bash_version=`$SH_PROG --version`
 [bash_minor=`$SH_PROG -c 'echo ${BASH_VERSINFO[1]}'`]
 bash_5_or_greater=no
 case "${bash_major}.${bash_minor}" in
-  'OK_BASH_VERS' | '5.0' | '5.1')
+  'OK_BASH_VERS' | '5.0' | '5.1' | '5.2')
     bash_5_or_greater=yes
     ;;
   *)
